management_interface/internal_integrations/management_api/client.py

```diff
@@ -0,0 +1,58 @@
+import uuid
+from datetime import date
+from typing import Optional
+
+import requests
+from fhir.resources.operationoutcome import OperationOutcome
+
+from internal_integrations.management_api.exceptions import ManagementAPIClientError
+
+from internal_integrations.management_api.settings import get_management_api_settings
+
+
+class ManagementAPIClient:
+    def __init__(self, base_url: Optional[str] = None, session: Optional[requests.Session] = None):
+        self.base_url = base_url or get_management_api_settings().base_url
+        self.session = session or requests.Session()
+
+    def create_subscription(self, patient_given_name: str, patient_family_name: str, nhs_number: str,
+                            birth_date: date) -> uuid.UUID:
+        body = {
+            "resourceType": "Patient",
+            "identifier": [
+                {
+                    "system": "https://fhir.nhs.uk/Id/nhs-number",
+                    "value": nhs_number,
+                    "extension": [
+                        {
+                            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
+                            "valueCodeableConcept": {
+                                "coding": [
+                                    {
+                                        "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-NHSNumberVerificationStatusEngland",
+                                        "code": "03",
+                                        "display": "Trace required"
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ],
+            "name": [
+                {
+                    "use": "official",
```

Comment:
Can we change this to "usual" - it's my mistake in the API specification but just want to keep consistent throughout as we expect hospital and PDS to send us things with "usual".
```suggestion
                    "use": "usual",
```